test/spec/modules/vertamediaBidAdapter_spec.js

```diff
@@ -42,72 +107,115 @@ describe('vertamediaBidAdapter', () => {
+    let videoBidRequests = [VIDEO_REQUEST];
+    let dispalyBidRequests = [DISPLAY_REQUEST];
 
-    const request = spec.buildRequests(bidRequests, {});
+    const displayRequest = spec.buildRequests(dispalyBidRequests, {});
+    const videoRequest = spec.buildRequests(videoBidRequests, {});
 
     it('sends bid request to ENDPOINT via GET', () => {
-      expect(request[0].method).to.equal('GET');
+      expect(videoRequest[0].method).to.equal('GET');
+      expect(displayRequest[0].method).to.equal('GET');
     });
+
     it('sends bid request to correct ENDPOINT', () => {
-      expect(request[0].url).to.equal(ENDPOINT);
+      expect(videoRequest[0].url).to.equal(ENDPOINT);
+      expect(displayRequest[0].url).to.equal(ENDPOINT);
     });
 
-    it('sends correct bid parameters', () => {
-      const bid = Object.assign({}, request[0].data);
+    it('sends correct video bid parameters', () => {
+      const bid = Object.assign({}, videoRequest[0].data);
       delete bid.domain;
 
       const eq = {
         callbackId: '84ab500420319d',
+        ad_type: 'video',
         aid: 12345,
         w: 640,
         h: 480
       };
 
       expect(bid).to.deep.equal(eq);
     });
+
+    it('sends correct display bid parameters', () => {
+      const bid = Object.assign({}, displayRequest[0].data);
+      delete bid.domain;
+
+      const eq = {
+        callbackId: '84ab500420319d',
+        ad_type: 'display',
+        aid: 12345,
+        w: 300,
+        h: 250
+      };
+
+      expect(bid).to.deep.equal(eq);
+    });
   });
 
   describe('interpretResponse', () => {
-    let bidderRequest = {
-      bidderCode: 'bidderCode',
-      bids: [{mediaTypes: {video: {}}}]
-    };
+    let serverResponse;
+    let bidderRequest;
+    let eqResponse;
+
+    afterEach(() => {
+      serverResponse = null;
+      bidderRequest = null;
+      eqResponse = null;
+    });
+
+    it('should get correct video bid response', () => {
+      serverResponse = SERVER_VIDEO_RESPONSE;
+      bidderRequest = videoBidderRequest;
+      eqResponse = videoEqResponse;
+
+      bidServerResponseCheck();
+    });
+
+    it('should get correct display bid response', () => {
+      serverResponse = SERVER_DISPLAY_RESPONSE;
+      bidderRequest = displayBidderRequest;
+      eqResponse = displayEqResponse;
 
-    it('should get correct bid response', () => {
+      bidServerResponseCheck();
+    });
+
+    function bidServerResponseCheck() {
       const result = spec.interpretResponse({body: serverResponse}, {bidderRequest});
-      const eq = [{
-        vastUrl: 'http://rtb.vertamedia.com/vast/?adid=44F2AEB9BFC881B3',
-        requestId: '2e41f65424c87c',
-        creativeId: 342516,
-        mediaType: 'video',
-        netRevenue: true,
-        currency: 'USD',
-        height: 480,
-        width: 640,
-        ttl: 3600,
-        cpm: 0.9
-      }];
-
-      expect(result).to.deep.equal(eq);
+      console.log(result);
+      console.log(eqResponse);
```

Comment:
Please drop the above two `console.log` statements

Reply:
@matthewlane 
Code has been updated
Please take a look.